src/app/test-record/emission-details/emission-details.component.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { EmissionDetailsComponent } from './emission-details.component';
+import {SharedModule} from '@app/shared/shared.module';
+import {TechRecordHelpersService} from '@app/technical-record/tech-record-helpers.service';
+import {TestType} from '@app/models/test.type';
+import {TestResultModel} from '@app/models/test-result.model';
+
+describe('EmissionDetailsComponent', () => {
+  let component: EmissionDetailsComponent;
+  let fixture: ComponentFixture<EmissionDetailsComponent>;
+  const testType = {} as TestType;component
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [ SharedModule ],
+      providers: [
+        TechRecordHelpersService
```

Comment:
Mock this service. otherwise during test execution the real service would be called. We dont want that